### ISSUE TITLE
PlotSignal with DateTime X-axis / Middle mouse zoom wrong result

### ISSUE THREAD

**Issue by bukkideme** [swharden/ScottPlot#351]
**Describe the bug**
When using the middle mouse "rectangle" zoom on a PlotSignal WinForm plot, the result shows empty graph, with no ticks and labels on X-axis. When using the right mouse click zoom options, everything works as expected.

**To Reproduce**
Create a Winform PlotSignal plot, and use the middle mouse zoom.

Some relevant code parts:
`formsPlot1.plt.YLabel("Sensor 1 Temperature (°C)");`
           ` formsPlot1.plt.Ticks(dateTimeX: true);`
            `formsPlot1.plt.AxisAuto();`
           ` formsPlot1.plt.AntiAlias(true, true);`
`formsPlot1.plt.PlotSignal(temp1Data.ToArray(), sampleRate: rate, xOffset: timeOffset.ToOADate(), color: Color.Blue, lineWidth: 2, markerSize: 6);`
`formsPlot1.Render();`

![ZoomRectangle_middleMouseButton](https://user-images.githubusercontent.com/51174971/79578585-b5312c00-80c6-11ea-96c4-12c4d6d30523.png)

![ZoomRectangle_middleMouseButton_Result](https://user-images.githubusercontent.com/51174971/79578619-bf532a80-80c6-11ea-9200-7202e919e9ff.png)

**Comment by swharden** [comment]
Thanks for mentioning this @bukkideme!

I created a new demo to help test/monitor this behavior. I found this error does not affect all DateTime axes, just ones that are highly zoomed in (displaying seconds/minutes).

This bug relates to #269. This error is caused by the conversion between `float` and `double`. DateTime objects which track seconds/minutes require double precision, but mouse the tracking code is probably relying on PointF (float precision) . I will fix this shortly by ensuring the middle-click-drag mouse tracker uses doubles everywhere 👍
